test/e2e/test/apmserver/steps_init.go

```diff
@@ -21,15 +20,15 @@ func (b Builder) InitTestSteps(k *test.K8sClient) test.StepList {
 			Name: "K8S should be accessible",
 			Test: func(t *testing.T) {
 				pods := corev1.PodList{}
-				err := k.Client.List(&client.ListOptions{}, &pods)
+				err := k.Client.List(&pods)
```

Comment:
What namespace is it going to use if there's no namespace specified in the variadic option args?

Reply:
It will not filter on namespace, so it will just list all pods here. This looks like the same behavior as it was previously since the ListOptions obj was empty.